### ISSUE TITLE
Question on quote escaping

### ISSUE THREAD

**Issue by itscaro** [PowerShell/PowerShell#6935]
Is it correct that in PowerShell this is not correct way to escape double quote:

```
docker image inspect 4a89600a6cab -f '{{index .Config.Labels "key"}}'
```

I need to escape like below in order to make it work

```
docker image inspect 4a89600a6cab -f "{{index .Config.Labels \`"key\`"}}"
```

**Comment by mklement0** [comment]
Yes, extra escaping is - unfortunately - required, and for the sake of backward compatibility that won't change.

In short: you must _additionally_ `\`-escape `"` chars. you pass to external executables.

While your fix works, there is no strict need to switch to `"..."` from `'...'`, however; the following should do as well:

```powershell
docker image inspect 4a89600a6cab -f  '{{index .Config.Labels \"key\"}}'
```

For the full story, see https://github.com/PowerShell/PowerShell-Docs/issues/2361
